include/swift/AST/Decl.h

```diff
@@ -6703,7 +6703,9 @@ inline bool ValueDecl::isImportAsMember() const {
 inline bool Decl::isPotentiallyOverridable() const {
   if (isa<VarDecl>(this) ||
       isa<SubscriptDecl>(this) ||
-      isa<FuncDecl>(this)) {
+      isa<FuncDecl>(this) ||
+      isa<AccessorDecl>(this) ||
```

Comment:
AccessorDecl is a FuncDecl

Reply:
Oops, I thought it was just an AbstractFunctionDecl. Thanks.